src/gui/raster/qgssinglebandpseudocolorrendererwidget.cpp

```diff
@@ -363,6 +363,11 @@ void QgsSingleBandPseudoColorRendererWidget::on_mClassifyButton_clicked()
   double min = lineEditValue( mMinLineEdit );
   double max = lineEditValue( mMaxLineEdit );
 
+  if ( qIsNaN( min ) || qIsNaN( max ) )
```

Comment:
the reason to use qt method instead of the std one? http://en.cppreference.com/w/cpp/numeric/math/isnan ?
should be coerent with the std NaN returned from lineEditValue:
https://github.com/boundlessgeo/QGIS/blob/50b663ead4acaf945799c3989923353a9848b091/src/gui/raster/qgssinglebandpseudocolorrendererwidget.cpp#L941
Any opinion?

Reply:
AFAIK for master we are using std, but for 2.x I've just used the same macro that is used through all the rest of the class.